pkg/distributor/distributor.go

```diff
@@ -125,6 +125,7 @@ type Config struct {
 
 	HATrackerConfig HATrackerConfig `yaml:"ha_tracker,omitempty"`
 
+	MaxRecvMsgSize      int           `yaml:"max_send_msg_size"`
```

Comment:
The yaml here is `max_send_msg_size`, while the CLI flag is `max-recv-msg-size`. Isn't contradictory?

Reply:
Good catch, could you open a PR/issue so that we don't lose track of this?

Reply:
oops, sorry!